### DISCUSSION TITLE
Problem with an Apple II+

### DISCUSSION THREAD

**Discussion by JamieB226** [markadev/AppleII-VGA#42] [Q&A] [answered]
I built the VGA board and made a custom cable to connect it to TV.  I have done the power on self test using the board outside of the computer and it works as described (Inverse @ signs that look crisp).  When I install the card in the computer (Apple II+ without any other cards), I get a pattern that is very similar to the test pattern but with a few of the @ signs replaced with other characters.  It looks nothing like the "Apple ][" logo and prompt I should be seeing.  If I connect the composite output to my adapter I get a very grainy version of the expected logo and prompt.  I'm looking for any advice or next steps to troubleshoot this. Suggestions?

Please see pictures of the self test, the composite out, and the VGA board output when in the computer.
![20240402_170953](https://github.com/markadev/AppleII-VGA/assets/7917779/e3adbe46-df4c-4aa5-9bfd-eca9b27845ed)
![20240402_171051](https://github.com/markadev/AppleII-VGA/assets/7917779/1e3b3e0d-be79-413c-bd53-6e61e4ba0d5a)
![20240402_170543](https://github.com/markadev/AppleII-VGA/assets/7917779/1cb29e97-6ac8-4dee-8e6d-ace7052c3cf8)

**Comment by markadev** [comment, marked as answer]
Hey there, it seems like the signals between the Apple II bus and the Pico may have a problem here.  Can you post a picture of your card too?

You can try a few troubleshooting steps like:
 * using a multimeter to verify the connections from the bus connector to ICs U1-U4 and ICs U1-U4 to the Pico.  You can check [the schematic](https://github.com/markadev/AppleII-VGA/blob/main/AppleVGA/AppleVGA.pdf) to see which IC pins are supposed to be connected.
 * double-check that you are using "74LVC245" chips and not another variant like "LS"
 * If you have a scope then you can check that the signal on pin 31 of the Pico should be a clean 3.3v square wave of ~1MHz

**Reply by JamieB226** [reply]
I got all the parts from your Mouser links.  So, the 245's were the correct LVC version.  I tested all the connections from the slot connector to the 245's and they were good. The square wave was a little funny.  Lots of frequency fluctuations.  So, I started checking continuity between the pi and the board.  Some definite issues there.  I pulled it out and reseated it and everything started to work.  Thanks for the suggestions.  At least it was an easy fix.
